packages/zowe-explorer/src/shared/init.ts

```diff
@@ -116,34 +116,28 @@ export async function registerCommonCommands(context: vscode.ExtensionContext, p
             )
         );
         context.subscriptions.push(
-            vscode.workspace.onWillSaveTextDocument(async (savedFile) => {
+            vscode.workspace.onDidSaveTextDocument((savedFile) => {
                 globals.LOG.debug(
                     localize(
                         "onDidSaveTextDocument1",
                         "File was saved -- determining whether the file is a USS file or Data set.\n Comparing (case insensitive) "
                     ) +
-                        savedFile.document.fileName +
+                        savedFile.fileName +
                         localize("onDidSaveTextDocument2", " against directory ") +
                         globals.DS_DIR +
                         localize("onDidSaveTextDocument3", "and") +
                         globals.USS_DIR
                 );
-                if (!savedFile.document.isDirty) {
```

Comment:
Just for my personal understanding, we are adding non-dirty files to the queue, right?

Reply:
That's right, if a save event is triggered by VS Code for a non-dirty file, we will process it.